eos/models/wilson-coefficients.hh

```diff
@@ -37,6 +37,11 @@ namespace eos
     {
         struct ClassII {};  // |Delta B| = 1 semileptonic operators, cf. [AFGV:2017A], eq. (2.5), p. 6.
         struct ClassIII {}; // |Delta B| = 1 = |Delta C| four-quark operators, cf. [AFGB:2017A], eq. (2.6), p. 7.
+        namespace ClassVhadronic
+        {
+            struct full {}; // |Delta B| = 1, |Delta C| = 0 four-quark operators sbqq (q=u,d,c), cf. [AFGB:2017A], eq. (2.10)-(2.11), p. 8.
+            struct restricted {}; // |Delta B| = 1, |Delta C| = 0 four-quark operators, sbqq (q=s,b) cf. [AFGB:2017A], eq. (2.10)-(2.12), p. 8.
```

Comment:
Just now Danny and I couldn't come up with anything better than `full` and `restricted` but maybe there is something else